.npmignore

```diff
@@ -1,33 +0,0 @@
-# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Seems preferable to me. I will be using package.json's `files` from now on, I think.